### ISSUE TITLE
Learning IMM Filter Parameters from Measurements using Gradient Descent. (arXiv:2307.06618v2 [cs.LG] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#29508]
Learning IMM Filter Parameters from Measurements using Gradient Descent. (arXiv:2307.06618v2 [cs.LG] UPDATED)<br>
https://ift.tt/h4W1oBI<br>
<p>The performance of data fusion and tracking algorithms often depends on
parameters that not only describe the sensor system, but can also be
task-specific. While for the sensor system tuning these variables is
time-consuming and mostly requires expert knowledge, intrinsic parameters of
targets under track can even be completely unobservable until the system is
deployed. With state-of-the-art sensor systems growing more and more complex,
the number of parameters naturally increases, necessitating the automatic
optimization of the model variables. In this paper, the parameters of an
interacting multiple model (IMM) filter are optimized solely using
measurements, thus without necessity for any ground-truth data. The resulting
method is evaluated through an ablation study on simulated data, where the
trained model manages to match the performance of a filter parametrized with
ground-truth values.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
